cfpb/cfgov-refresh
# Portal category sort

Add a `display_order` field to PortalCategory to control nav sequence on explore pages.

We want category choices to follow this order, to reflect the order
used on portal topic pages:
* Basics
* Key terms
* Common issues
* Know your rights
* How-to guides

To give editors control over the order, in case categories change
in the future, we're adding a `display_order` field that allows an editor
to assign a sort number to a portal category.

The migration of the new field includes a data migration that assigns
display_order to our current set of categories.

## Testing
- Pull in branch and a fresh db dump
- Migrate the model change:
```bash
./cfgov/manage.py migrate
```
- Make sure the sequence of categories remains as described above:   
<http://localhost:8000/consumer-tools/payday-loans/answers/common-issues/>
